main.py

```diff
@@ -0,0 +1,70 @@
+"""
+You could download multiple courses at once 
+with help of the script from stepic.org .
+
+
+Run by: 
+python3 main.py
+
+Additional notes:
+1) Add courses which you want to download to the "courses" dictionary.
+
+2) You could also set quality of the video by changing "quality" variable. 
+The default quality is 720.
+"""
+
+import os
+
+import weekDownloader
+
+def main():
```

Comment:
I tried to save functional backward compatibility with the old version, so you could pass args if you want to download one week or edit the file and download the whole course. 
I could delete the old main function if you want.